test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py

```diff
@@ -66,14 +66,28 @@ def test_performance_ec2_tensorflow_inference_graviton_cpu(
     )
 
 
+@pytest.mark.model("inception, RCNN-Resnet101-kitti, resnet50_v2, mnist, SSDResnet50Coco")
```

Comment:
is there a reason to add this? it is currently being skipped anyway, so potentially not worth adding

Reply:
yeah I debated it as well, but figured it's better to be consistent across the board in case we un-skip this test. If that's unlikely then I can remove it